Makefile

```diff
@@ -13,14 +13,14 @@ load-design-system-templates:
 	pipenv run ./scripts/load_templates.sh $(DESIGN_SYSTEM_VERSION)
 
 lint:
-	pipenv check ./response_operations_ui ./tests --ignore=70612
+	pipenv check ./response_operations_ui ./tests --ignore=70612 --ignore=73711
```

Comment:
Only added so this will build, there is another PR I have raised which will fix this

Reply:
I think we may be able to remove the other ignore as well and its not coming up when running the lint command

Reply:
I've just merged the other one, so will see if I can get rid of both, as well as adding in CODEOWNERS